net/strongswan/files/ipsec.init

```diff
@@ -245,7 +245,7 @@ config_remote() {
 
 	[ "$gateway" = "any" ] && remote_gateway="%any" || remote_gateway="$gateway"
 
-	[ -z "$local_identifier" ] && {
```

Comment:
Unclear what this has to do with the local_identifier.  Seems more related to the local_gateway.